lib/scrabble.rb

```diff
@@ -15,4 +27,27 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def prepare(word)
+    if word.class != Array
+      word.upcase!
+      word = word.chars
+    else
+      word
```

Comment:
Intrigued by this: was there a test that led to this? When would we ever pass this method an array? Even if we did, wouldn't we still want to run `#upcase` on each of the letters in the `word` array?